doc/lightning-getroute.7.txt

```diff
@@ -29,6 +29,19 @@ fail, and it would cost you 20% per annum if that happened,
 
 If you didn't care about risk, 'riskfactor' would be zero.
 
+The 'fuzzpercent' is a floating-point number, between 0.0 to 100.0,
+unit of percent.
+The 'fuzzpercent' is used to distort computed fees along each channel,
+to provide some randomization to the route generated.
+0.0 means the exact fee of that channel is used,
+while 100.0 means the fee used might be from 0 to twice the actual fee.
+The default is 5.0, or up to 5% fee distortion.
```

Comment:
The default is 75.0, or up to 75% fee distortion.

Reply:
Oops!